backend/controllers/adminController.js

```diff
@@ -0,0 +1,21 @@
+const User = require("../models/user");
+
+//Update role of a user
+function updateRole(req, res) {
+    User.update({
+        Role: req.body.role
+    }, {
+        where: {
+            Email: req.body.email
+        }
+    }).then(result => {
+        console.log("Role successfully updated !");
+        res.json("Role succesfully updated !");
```

Comment:
do you think a 200 should be returned also `res.status(200).json("Role succesfully updated !");` or is that not needed?

Reply:
Excellent idea !